go-ducks.js

```diff
@@ -1,4 +1,4 @@
-const consumables = [() => ''];
+const consumables = [() => ' SECRET'];
```

Comment:
@muralidhar-chintada It looks like there is a space before SECRET.  Please remove the space.